ide/db/src/org/netbeans/modules/db/explorer/node/ProcedureNodeProvider.java

```diff
@@ -202,8 +201,8 @@ public void run(Metadata metaData) {
                     NodeRegistry.handleMetadataModelException(this.getClass(), connection, e, true);
                 }
             }
-        } else if (connection != null && connection.getDriverName() != null &&
-                connection.getDriverName().startsWith(DatabaseModule.IDENTIFIER_ORACLE)) {
+        } else if (connection != null && connection.getDriverName() != null && connection.getDriverName().startsWith(DatabaseModule.IDENTIFIER_ORACLE)) {
```

Comment:
This PR deals with MySQL and the changes below are cosmetical - please remove them.